packages/nocodb/src/utils/tele.ts

```
@@ -248,6 +249,41 @@
+      package_id: packageVersion,
+      node_version: process.version,
+      xc_version: process.env.NC_SERVER_UUID,
+      env: process.env.NODE_ENV || 'production',
+      oneClick: !!process.env.NC_ONE_CLICK,
+    };
+    try {
+      payload.os_type = os.type();
+      payload.os_platform = os.platform();
+      payload.os_release = os.release();
+      payload.docker = isDocker();
+      payload.machine_id = `${this.id},,`;
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)